samples/snippets/fsharp/azure/queue-storage.fsx

```diff
@@ -12,102 +11,88 @@ let storageConnString = "..." // fill this in from your storage account
 let storageConnString = 
     CloudConfigurationManager.GetSetting("StorageConnectionString")
 *)
-//
-// Parse the connection string.
-//
-
-// Parse the connection string and return a reference to the storage account.
-let storageAccount = CloudStorageAccount.Parse(storageConnString)
 
 //
 // Create the Queue Service client.
 //
 
-let queueClient = storageAccount.CreateCloudQueueClient()
-
-//
-// Create a queue.
-//
-
-// Retrieve a reference to a container.
-let queue = queueClient.GetQueueReference("myqueue")
-
-// Create the queue if it doesn't already exist
-queue.CreateIfNotExists()
+let queueClient = new QueueClient(storageConnString, "myqueue");
```

Comment:
```suggestion
let queueClient = QueueClient(storageConnString, "myqueue");
```

Unless this is disposable, you can omit the `new`

Reply:
Thanks for your guidance!